components/dashboard/src/user-settings/Preferences.tsx

```diff
@@ -72,6 +88,37 @@ export default function Preferences() {
                         </p>
                     </div>
                 </div>
+
+                <h3 className="mt-12">Inactivity Timeout </h3>
+                <p className="text-base text-gray-500 dark:text-gray-400">
+                    By default, workspaces stop following 30 minutes without user input (e.g. keystrokes or terminal
+                    input commands). You can increase the workspace timeout up to a maximum of 24 hours.
+                </p>
+                <div className="mt-4 max-w-xl">
+                    <h4>Default Workspace Inactivity Timeout</h4>
+                    <span className="flex">
+                        <input
```

Comment:
**Suggestion:** If disabled, can we add the following helper text to explain: "_Only paid users can update the inactivity timeout. See [pricing](https://www.gitpod.io/pricing) for more_.", otherwise I expect users will ask us why it is disabled 🙏

Reply:
See proposal in https://github.com/gitpod-io/gitpod/pull/16503#discussion_r1122062145 and reference of this pattern in https://github.com/gitpod-io/gitpod/pull/16503#discussion_r1122056419. Cc @iQQBot @loujaybee 

Also, shortening the section description as described in https://github.com/gitpod-io/gitpod/pull/16503#discussion_r1122062145 can help. We can rely on the validation error to communicate max values and avoid nudging users to use 24h here. 💡